app/controllers/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+class UsersController < ApplicationController
+  def show
+    @favorites = []
+    # @favorites = fetch_favorites_for_user(current_user.id)
```

Comment:
looks good! I'll just change this empty bracket to a get request from the service to the backend